lib/active_admin/filters/active_filter.rb

```diff
@@ -26,19 +26,29 @@ def values
       end
 
       def label
+        translated_predicate = translated_predicate_name.mb_chars.downcase.to_s
```

Comment:
What's special about this situation? We're not handling multibyte chars specially anywhere else, right? And this works out of the box in ruby 2.4

```shell
$ ruby --version
ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-linux]
irb
irb(main):001:0> "ÀÄáâãäå".downcase
=> "àäáâãäå
```